components/process.rst

```diff
@@ -258,37 +258,13 @@ instead::
     );
     $process->run();
 
-To make your code work better on all platforms, you might want to use the
-:class:`Symfony\\Component\\Process\\ProcessBuilder` class instead::
+To make your code work better on all platforms, you might want to pass an array of
+arguments to the constructor instead::
 
-    use Symfony\Component\Process\ProcessBuilder;
-
-    $builder = new ProcessBuilder(array('ls', '-lsa'));
-    $builder->getProcess()->run();
-
-In case you are building a binary driver, you can use the
```

Comment:
Not sure about this removal but it looks like we can no longer register a prefix